modules/lambdas/variables.tf

```diff
@@ -121,6 +121,16 @@ variable "orca_recovery_retry_backoff" {
   description = "The multiplier by which the retry interval increases during each attempt."
 }
 
+variable "orca_s3_access_key" {
```

Comment:
Not presently set up. https://bugs.earthdata.nasa.gov/browse/ORCA-373

Reply:
Supposedly tf allows you to create an IAM user with these properties, but I haven't messed around with it yet.

Reply:
Unfortunately, we cannot create users https://wiki.earthdata.nasa.gov/display/ESKB/AWS+Services+Approval+Status+Page#AWSServicesApprovalStatusPage-IAM.1 it is wone of our limitations to meet security controls.